frontend/src/pages/invite/ui/inviteMessage.ts

```diff
@@ -0,0 +1,101 @@
+import { msg } from "@lit/localize";
+import { html, type TemplateResult } from "lit";
+import { when } from "lit/directives/when.js";
+
+import type { UserOrgInviteInfo } from "@/types/user";
+
+export const renderInviteMessage = (
+  inviteInfo: void | UserOrgInviteInfo,
+  { isExistingUser, isOrgMember } = {
+    isExistingUser: false,
+    isOrgMember: false,
+  },
+) => {
+  if (!inviteInfo) return;
+
+  let message: string | TemplateResult = "";
+
+  if (inviteInfo.firstOrgAdmin) {
+    if (isExistingUser) {
+      message = msg("Set up your organization to start crawling.");
+    } else {
+      message = msg(
+        "Finish setting up your Browsertrix account to start crawling.",
+      );
+    }
+  } else {
+    const { inviterName, orgName, fromSuperuser } = inviteInfo;
+
+    if (inviterName && !fromSuperuser && orgName) {
+      message = msg(
+        html`You’ve been invited by
+          <strong class="font-medium">${inviterName}</strong>
+          to join the organization
+          <span class="font-medium text-primary">${orgName}</span>
+          on Browsertrix.`,
+      );
+    } else if (orgName) {
+      message = msg(
+        html`You’ve been invited to join the organization
+          <span class="font-medium text-primary">${orgName}</span>
+          on Browsertrix.`,
+      );
+    }
+  }
+
+  if (!message) return;
+
+  return html`<p class="max-w-prose text-base text-neutral-700">${message}</p>
+    ${when(
+      inviteInfo.firstOrgAdmin,
+      () => html`
+        <ul class="mt-6 text-base text-neutral-700">
+          <li class="mb-3 flex items-center gap-2">
+            <sl-icon
+              class="text-lg text-primary"
```

Comment:
Can we make the logo (and maybe also the text) green when a step has been completed?
```suggestion
              class=${clsx(
                "text-lg",
                isOrgMember ? "text-success" : "text-primary",
              )}
```